### ISSUE TITLE
Feature request - launch program/script/arexx on start and quit 

### ISSUE THREAD

**Issue by kowoba** [koobo/HippoPlayer#12]
Path to program/script/arexx command that will be launched on startup and on quit.
Usecase - install a "HiP" menu in Workbench (via arexx on v45+) on startup and remove it again on quit.

**Comment by koobo** [comment]
You could probably do an ARexx script that
- waits for Hippo to start
- adds menu magic
- waits for Hippo to exit
- removes menu magic

**Comment by kowoba** [comment]
How do one wait for HiP to finish? Part of the problem is that HiP detaches itself, so the only way to "wait for exit" is to have script running in the background with the only purpose of regularly check if HiP is still running. Or did I miss something obvious?

**Comment by koobo** [comment]
Yeah, it should be an ARexx script that starts and polls for Hippo ARexx port to find out if it is running or not. This approach was used by a notifier application that displays the current module being played, I don't remember the name unfortunately. This app worked with some other module players, not Hippo, but the idea was similar.
